stripes/www/stripes_home.html

```diff
@@ -227,14 +217,52 @@ <h1 class="m-0 logo">Pollution Stripes</h2>
 			}
 		};
 
+		/* Fetching Stirpes */
+		const fetch_stripes = (value) => {
+			const stripes_container = document.querySelector(".stripes-container");
+			stripes_container?.classList.add("loading");
+
+			frappe.call({
+				method: 'stripes.svg.get_stripes_svg',
+				args: {
+					from_date: "2018-01-01",
+					to_date: "2018-12-31",
```

Comment:
Instead of hardcoding and writing the same date in multiple locations... make it a global window variable `window.stripes_from_date = "2018-01-01"` and `window.stripes_to_date = "2018-12-31"` and use the variables to commonify